tests/snapshots/init/auto_git.stderr

```diff
@@ -0,0 +1 @@
+     Created library package
```

Comment:
>  ehuss commented on the distance between the commands and tests

I totally agree with you. It's not that painful if only few tests being added at a time. Indeed there is a distance between commands and tests, though I feel like it gets some benefits, such as:

- Can tweak the input/output without re-compiling.
- People can get a real file layout instead of pondering in their head.

The code-defined fixture can be easily written in an issue description. This is one of the most missed feature IMO. Other thing like code review o, I'll recommend prepare a list of checkboxes of test cases and check them one by one. 

One distance I felt uncomfortable is the distance between `*.in`/`*.out` and `*.stderr`/`*.stdout`. Most IDE/file-manager list files in a directory-first way, and it causes the review harder.

Reply:
> One distance I felt uncomfortable is the distance between `*.in`/`*.out` and `*.stderr`/`*.stdout`. Most IDE/file-manager list files in a directory-first way, and it causes the review harder.

I thought this as well when working on #10606. It almost feels like everything should be in one or two folders and kept closer together.
```
foo_test/
    in/
    out/
    stderr
    stdout
```

Reply:
> One distance I felt uncomfortable is the distance between *.in/*.out and *.stderr/*.stdout. Most IDE/file-manager list files in a directory-first way, and it causes the review harder.

I'm used to `ls` where I get them in alphabetical order.

I was also thinking to move everything into the subdirectory.  Would you prefer me to do it in master before we merge this PR and you update the PR or should we update the style of both after we merge this?  
(I lean towards the latter)

Reply:
https://github.com/rust-lang/cargo/issues/10626